src/engine/output-results.ts

```diff
@@ -12,7 +12,8 @@ import {
   CriteriaResultItem,
   Documentation
 } from 'pathways-model';
-import { DocumentReference, DomainResource } from 'fhir-objects';
+import { DomainResource } from 'fhir-objects';
```

Comment:
I don't think DocumentReference should have been removed here. I'm getting an error when I run this because it is used on line 235
```suggestion
import { DocumentReference, DomainResource } from 'fhir-objects';
```